test/e2e/test_item.py

```diff
@@ -82,12 +84,55 @@
 }
 
 
+# pylint: disable=duplicate-code
+def _post_units(test_client):
+    """Utility function for posting all mock units defined at the top of this file"""
+
+    response = test_client.get("/v1/units")
+    units = response.json()
+
+    if response.json() == []:
+
+        response = test_client.post("/v1/units", json=UNIT_POST_A)
+        unit_mm = response.json()
+
+        response = test_client.post("/v1/units", json=UNIT_POST_B)
+        unit_cm = response.json()
+
+        units = [unit_mm, unit_cm]
+
+    return units
+
+
+def _post_catalogue_category_with_units(test_client, catalogue_category):
+    """Utility function for posting Catalogue category a defined at the top of this file"""
+
+    units = _post_units(test_client)
+
+    response = test_client.post(
+        "/v1/catalogue-categories",
+        json={
+            **catalogue_category,
+            "catalogue_item_properties": replace_unit_values_with_ids_in_properties(
+                catalogue_category["catalogue_item_properties"], units
+            ),
+        },
+    )
+    catalogue_category = response.json()
+
+    return catalogue_category
+
+
+# pylint: enable=duplicate-code
+
+
 def test_create_item(test_client):
     """
     Test creating an item.
     """
-    response = test_client.post("/v1/catalogue-categories", json=CATALOGUE_CATEGORY_POST_A)
-    catalogue_category = response.json()
+    # pylint: disable=duplicate-code
```

Comment:
```suggestion
    # pylint: disable=duplicate-code
```
Is this still needed, if so the one below should probably be removed.